cinema/views.py

```diff
@@ -56,3 +100,67 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset.select_related("movie")
+
+        if self.action == "list":
+            queryset = (
```

Comment:
This is a bit of a weird hyphenation, I think it would be better to do it like this
queryset = queryset.annotate(
...
)